### DISCUSSION TITLE
What is the changelog of release 2.0 (relative to 1.1)?

### DISCUSSION THREAD

**Discussion by orpheuslummis** [WebAssembly/spec#1451] [Q&A] [answered]
If there are notable changes, it would be good to indicate them.
https://github.com/WebAssembly/spec/pull/1443

**Comment by rossberg** [comment, marked as answer]
There is no difference, we simply had to rename the (tentative) version number for process reasons. See [my comment here](https://github.com/WebAssembly/design/issues/1395#issuecomment-1109606495) for some background.
